### PR TITLE
Include deployment name in Spack and Ramble bucket names (like startup-script)

### PR DESCRIPTION
**Pull request by rohitramu**
This will make it easier to know which buckets are tied to a deployment.

### CODE REVIEW

File: community/modules/scripts/ramble-setup/main.tf
Comment (nick-stroud) [inline community/modules/scripts/ramble-setup/main.tf:1]: Can you trim the deployment name to make sure length of bucket name does not exceed limit. I think it is 63 chars.

Comment (nick-stroud) [inline community/modules/scripts/ramble-setup/main.tf:1]: Please run spack tests with green before merging.

**Review by nick-stroud — APPROVED** [review]

File: community/modules/scripts/ramble-setup/main.tf
Comment (rohitramu) [reply community/modules/scripts/ramble-setup/main.tf:1]: Good point - done.

Doc for context: https://cloud.google.com/storage/docs/buckets#naming

Comment (rohitramu) [reply community/modules/scripts/ramble-setup/main.tf:1]: The `spack-gromacs` test has now succeeded.

**Pull request merged**